### ISSUE TITLE
How to know when to stop checking order status

### ISSUE THREAD

**Issue by artyom-v** [xsolla/store-unity-sdk#69]
I'm using XsollaCatalog.Purchase method for items purchasing.
How to know if user don't start paying in purchase UI and closed it, so I can stop tracking order status?

**Comment by DenisRakhimovXsolla** [comment]
The XsollaCatalog.Purchase method has an `onBrowseClosed` callback that will be called after the browser is closed and the `BrowserCloseInfo` data will be returned with the `isManually` variable. If this variable is true, this means that the player manually closed the browser and refuse the purchase.
But this only applies to the “in-game browser”.
Otherwise, we have no way to track user behavior.

**Comment by artyom-v** [comment]
Not true. Some of our users use mobile payments to purchase products. And they may close the "in-game browser" before the payment is successful. So onBrowseClosed callback is useless.

**Comment by DenisRakhimovXsolla** [comment]
in this case, you can start the process of getting the order status yourself - call the `XsollaOrders.CheckOrderStatus` method after closing the browser (ignoring the isManually variable).
